### ISSUE TITLE
th cells with pseudo-element ":after" with "content" css property produces duplication

### ISSUE THREAD

**Issue by guglielmobossi** [dompdf/dompdf#1518]
If the table appear in more than one page and the table head has **th** cells with a span with pseudo-element ":after" with "content" css property (code example attached) this case produce a duplication of the text like images (multiplied for every page)

It seems that for every page, the "content" css propery is applied another time.

code:
```
<style>
                .special-class1:after{content:"Ø nominale"}
             
</style>...
```

```
<th class="rotate ">
  <div>
    <span class="special-class1"></span>
  </div>
</th>
```

page 1:
![schermata 2017-08-21 alle 14 09 21](https://user-images.githubusercontent.com/4170756/29528847-c47c1538-869d-11e7-80a9-33fe94a4013d.png)

page 2:
![schermata 2017-08-21 alle 14 09 30](https://user-images.githubusercontent.com/4170756/29528846-c47bbeee-869d-11e7-9f3b-21ea9a4def37.png)

page 3:
![schermata 2017-08-21 alle 14 09 43](https://user-images.githubusercontent.com/4170756/29528848-c4896e18-869d-11e7-8eb9-c8c63a3b4238.png)

**Comment by bsweeney** [comment]
I believe this should be fixed by commit 7c3fe91.

**Comment by guglielmobossi** [comment]
Thank you very much Brian! The problem is solved!
